src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -145,4 +155,15 @@ public String getPath() {
         return path.toString();
     }
 
+    /**
+     * Verifies that the storage file exists on the disk.
+     * 
+     * @throws StorageFileNotFoundException if the storage file is missing.
+     */
+    public void checkIfExistsOnDisk() throws StorageFileNotFoundException {
+        if (!Files.exists(path)) {
+            throw new StorageFileNotFoundException("Storage file is missing");
+        }
```

Comment:
This is a mistake, should have returned true or false instead of using exceptions in this case. The exception should be used in other methods where file not found is an unusual case.